src/crc32c/__init__.py

```diff
@@ -42,3 +44,57 @@ def value(chunk):
         int: New CRC checksum computed for ``chunk``.
     """
     return crc32c._crc32c_cffi.lib.crc32c_value(chunk, len(chunk))
+
+
+class Checksum(object):
+    """Hashlib-alike helper for CRC32C operations."""
+    __slots__ = ('_crc',)
```

Comment:
Can you run `black --line-length=79 $(git ls-files -- '*.py')`?

Reply:
I can live with some of its choices (`"`, meh), but reformatting the table from the RFC?  What is the comment syntax to bail out of that?

Reply:
There is no bail-out for `black`. It's purpose is to avoid such arguments.